e2e/tests/list.js

```diff
@@ -4,53 +4,64 @@ import driver from '../chromeDriver';
 import listPageFactory from '../pages/ListPage';
 
 describe('List Page', () => {
-    const ListPage = listPageFactory('http://localhost:8083/#posts')(driver);
+    const ListPagePosts = listPageFactory('http://localhost:8083/#posts')(driver);
+    const ListPageComments = listPageFactory('http://localhost:8083/#comments')(driver);
 
-    beforeEach(async () => await ListPage.navigate());
+    before(async () => await ListPagePosts.navigate());
 
     describe('Pagination', () => {
         it('should display paginated list of available posts', async () => {
-            assert.equal(await ListPage.getNbPagesText(), '1-10 of 13');
+            assert.equal(await ListPagePosts.getNbPagesText(), '1-10 of 13');
         });
 
         it('should switch page when clicking on previous/next page buttons or page numbers', async () => {
-            await ListPage.nextPage();
-            assert.equal(await ListPage.getNbPagesText(), '11-13 of 13');
+            await ListPagePosts.nextPage();
+            await ListPagePosts.checkPagination('11-13 of 13');
```

Comment:
There is no `assert` in `checkPagination`. It seems that you expect a timeout for this test to fail, which is not a good practice IMO. You should replace `checkPagination` by `getPagination` and do the assert here (as it was before).